### DISCUSSION TITLE
[Auth app] Add a quick way to access the Auth app from other applications

### DISCUSSION THREAD

**Discussion by ungular** [ente-io/ente#2579] [Enhancements]
Suggestions: 
- Picture-in-picture popup button
- Notification button/tile
- Widget

**Comment by rom4ster** [comment]
One nice thing would be if the app could register as a smart home control. Those pop up in many fun places. Or if a quick settings tile for the app can be created.

**Reply by AminGamer08** [reply]
yes, now that iOS 18 is out, on iOS every app can create their own control center control, in whichever size they want, and users can put all of these (only the 1x1 sized ones of course) on the lockscreen shortcuts

**Reply by rom4ster** [reply]
> yes, now that iOS 18 is out, on iOS every app can create their own control center control, in whichever size they want, and users can put all of these (only the 1x1 sized ones of course) on the lockscreen shortcuts

Lol sure but no need to care what ios does, if it does not work there android always has this.

**Comment by Miltora** [comment]
Is there any updates on this?
